backend/src/v5/processors/teamspaces/projects/models/commons/settings.js

```diff
@@ -0,0 +1,37 @@
+/**
+ *  Copyright (C) 2024 3D Repo Ltd
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU Affero General Public License as
+ *  published by the Free Software Foundation, either version 3 of the
+ *  License, or (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU Affero General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Affero General Public License
+ *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+const { getAccessibleJobs } = require('../../../../../models/jobs');
+const { getMembers } = require('../../../../../models/modelSettings');
+const { getProjectAdmins } = require('../../../../../models/projectSettings');
+const { getTeamspaceAdmins } = require('../../../../../models/teamspaceSettings');
+
+const Settings = {};
+
+Settings.getMembers = async (teamspace, project, model, excludeViewers) => {
```

Comment:
rename too. members is misleading as there is no such concept as members to a model.

Ultimately you're trying to get a list of userse with access to the model.